### PR TITLE
Update ember packages

### PR DESCRIPTION
**Pull request by Yelinz**

### CODE REVIEW

**Comment by Yelinz** [comment]
Remove all traces of ember-native-dom-helpers

File: tests/integration/components/sy-datepicker/component-test.js [hunk 1]
```diff
@@ -127,6 +127,5 @@ describe('Integration | Component | sy datepicker', function() {
 
     // hide if focus changed into another element
     await blur('input')
-    expect(find('.sy-datepicker')).to.not.be.ok
```
Comment (czosel) [inline tests/integration/components/sy-datepicker/component-test.js]: This is not exactly a "fix", is it? :wink:

File: app/components/sy-datepicker/component.js [hunk 2]
```diff
@@ -23,11 +23,10 @@ export default Component.extend({
 
   actions: {
     handleBlur(dd, e) {
-      if (
-        !document
-          .getElementById(`ember-basic-dropdown-content-${dd.uniqueId}`)
-          .contains(e.relatedTarget)
-      ) {
+      let element = document.getElementById(
```
Comment (czosel) [inline app/components/sy-datepicker/component.js]: `const`

File: tests/acceptance/tour-test.js [hunk 3]
```diff
@@ -36,38 +41,14 @@ describe('Acceptance | tour', function() {
...
-
-  it('can ignore tour permanently', async function() {
-    await visit('/')
-
-    await click('.btn-never')
-
-    await visit('/someotherroute')
-    await visit('/')
-
-    expect(find('.modal--visible')).to.not.be.ok
```
Comment (czosel) [inline tests/acceptance/tour-test.js]: Why remove those tests?

File: tests/integration/components/report-row/component-test.js [hunk 4]
```diff
@@ -1,32 +1,24 @@
...
-  setupRenderingTest()
-
-  beforeEach(function() {
-    this.server = startMirage()
-  })
-
-  afterEach(function() {
-    this.server.shutdown()
-  })
+  let app = setupRenderingTest()
```
Comment (czosel) [inline tests/integration/components/report-row/component-test.js]: `const`?

File: tests/integration/components/progress-tooltip/component-test.js [hunk 5]
```diff
@@ -1,27 +1,20 @@
...
-import { startMirage } from 'timed/initializers/ember-cli-mirage'
+import setupMirage from 'ember-cli-mirage/test-support/setup-mirage'
 import EmberObject from '@ember/object'
-import wait from 'ember-test-helpers/wait'
 import moment from 'moment'
-import { find } from 'ember-native-dom-helpers'
 
 describe('Integration | Component | progress tooltip', function() {
-  setupRenderingTest()
+  let app = setupRenderingTest()
```
Comment (czosel) [inline tests/integration/components/progress-tooltip/component-test.js]: `const`

File: tests/integration/components/task-selection/component-test.js [hunk 6]
```diff
@@ -25,15 +24,8 @@ const TASK = EmberObject.create({
...
-  setupRenderingTest()
-
-  beforeEach(function() {
-    this.server = startMirage()
-  })
-
-  afterEach(function() {
-    this.server.shutdown()
-  })
+  let app = setupRenderingTest()
```
Comment (czosel) [inline tests/integration/components/task-selection/component-test.js]: `const`

File: tests/integration/components/user-selection/component-test.js [hunk 7]
```diff
@@ -16,15 +15,8 @@ const USER = EmberObject.create({
...
-  setupRenderingTest()
-
-  beforeEach(function() {
-    this.server = startMirage()
-  })
-
-  afterEach(function() {
-    this.server.shutdown()
-  })
+  let app = setupRenderingTest()
```
Comment (czosel) [inline tests/integration/components/user-selection/component-test.js]: `const`

File: mirage/factories/project.js [hunk 8]
```diff
@@ -4,7 +4,7 @@ import { randomDuration } from '../helpers/duration'
 export default Factory.extend({
   name: () => faker.commerce.productName(),
   estimatedTime: () => randomDuration(),
-  reviewers: hasMany('user'),
+  // reviewers: hasMany('user'),
```
Comment (czosel) [inline mirage/factories/project.js]: Why comment this out? If it shouldn't be there, maybe rather remove it?

File: tests/acceptance/tour-test.js (hunk 3 above)
Comment (Yelinz) [reply tests/acceptance/tour-test.js]: The used addon ember-site-tour changed and removed the later button.

**Pull request closed without merging**